### PR TITLE
added the mail-id of developers

### PR DESCRIPTION
**Pull request by Diksha2307**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 Project for 7th sem project
 Developed by
 Diksha Singh, Harsh Karn, Kenguva Sampath
+Email-id of  developers
```
Comment (karnHarsh) [inline README.md]: change this line to just 'contact us'

**Review by karnHarsh — CHANGES_REQUESTED** [review]
Please make requested changes

File: README.md (hunk 1 above)
Comment (Diksha2307) [reply README.md]: Done

**Review by karnHarsh — APPROVED** [review]
LGTM

**Pull request merged**